core/src/main/resources/reference.conf

```diff
@@ -6,6 +6,9 @@ freki.core {
   auto_create_tagks = false
   auto_create_tagvs = false
 
+  # The LabelClientType cache size
+  label_client_cache_size = 200000
```

Comment:
Could you make it so we can configure this based on label type and on whether it is the forward or reverse cache?